streamlit_folium/__init__.py

```diff
@@ -1,79 +1,26 @@
-import streamlit.components.v1 as components
-import folium
-from folium import plugins
+import hashlib
+import os
+import re
+
 import branca
+import folium
+import folium.plugins
+import streamlit.components.v1 as components
+from folium.utilities import normalize
+from jinja2 import UndefinedError
```

Comment:
Is jinja tagging along with folium? Just wondering if we need to specify it as a dependency

Reply:
Yes, it is. Folium uses Jinja2 templates in each of its classes to generate the js that it ultimately produces.